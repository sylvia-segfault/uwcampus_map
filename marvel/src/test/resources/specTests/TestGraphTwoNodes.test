# Test the parser on a graph that contains two nodes

LoadGraph twoNodeGraph TwoNodes.tsv
ListNodes twoNodeGraph
ListChildren twoNodeGraph HUMAN
ListChildren twoNodeGraph GORILLA-MAN