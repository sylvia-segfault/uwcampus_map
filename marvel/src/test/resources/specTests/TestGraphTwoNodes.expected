# Test the parser on a graph that contains two nodes

loaded graph twoNodeGraph
twoNodeGraph contains: GORILLA-MAN HUMAN
the children of HUMAN in twoNodeGraph are: GORILLA-MAN(AVF 4)
the children of GORILLA-MAN in twoNodeGraph are: HUMAN(AVF 4)